### [CVE-2016-9949](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2016-9949)
![](https://img.shields.io/static/v1?label=Product&message=n%2Fa&color=blue)
![](https://img.shields.io/static/v1?label=Version&message=n%2Fa&color=blue)
![](https://img.shields.io/static/v1?label=Vulnerability&message=n%2Fa&color=brighgreen)

### Description

An issue was discovered in Apport before 2.20.4. In apport/ui.py, Apport reads the CrashDB field and it then evaluates the field as Python code if it begins with a "{". This allows remote attackers to execute arbitrary Python code.

### POC

#### Reference
- https://bugs.launchpad.net/apport/+bug/1648806
- https://github.com/DonnchaC/ubuntu-apport-exploitation
- https://www.exploit-db.com/exploits/40937/

#### Github
No PoCs found on GitHub currently.

